olmo/train.py

```diff
@@ -352,7 +352,7 @@ def load_trainer_state_dict(self, state_dict: Dict[str, Any]) -> None:
         ]
 
         # Dataset / dataloader position.
-        checkpoint_epoch = state_dict.get("epoch", 0)
+        checkpoint_epoch = state_dict.get("epoch") or 0
```

Comment:
Now it will error out if the state_dict does not have the `epoch` key right? Will it always be present going forward?

Reply:
`.get()` won't throw a `KeyError` so I think it's okay